packages/rpc-provider/src/ws/index.ts

```diff
@@ -268,13 +280,7 @@ export class WsProvider implements ProviderInterface {
    * @description Returns the connection stats
    */
   public get stats (): ProviderStats {
-    return {
-      active: {
-        requests: Object.keys(this.#handlers).length,
-        subscriptions: Object.keys(this.#subscriptions).length
-      },
```

Comment:
We lose this information - which is actually the only useful as-is information, current connections. (This is due to the rework of the structure above). We are actually interested in this, not just historic.